src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerActor.scala

```diff
@@ -35,7 +35,10 @@ object InstanceTrackerActor {
   private[impl] case class Get(instanceId: Instance.Id)
 
   /** Forward an update operation to the child [[InstanceUpdateActor]]. */
-  private[impl] case class ForwardTaskOp(deadline: Timestamp, instanceId: Instance.Id, op: InstanceUpdateOperation)
+  private[impl] case class UpdateContext(deadline: Timestamp, op: InstanceUpdateOperation) {
```

Comment:
Much better name but I'm not sure the docs are still good 😉

Reply:
➕ 1